frontend/test/projects/projects.store.spec.ts

```diff
@@ -0,0 +1,84 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { ProjectsActions } from '@/v5/store/projects/projects.redux';
+import { selectCurrentProject, selectCurrentProjectDetails, selectCurrentProjects } from '@/v5/store/projects/projects.selectors';
+import { TeamspacesActions } from '@/v5/store/teamspaces/teamspaces.redux';
+import { times } from 'lodash';
+import { projectMockFactory } from './projects.fixtures';
+import { createTestStore } from '../test.helpers';
+
+const projectListIncludesProject = (projectList, project) => (
+	projectList.map(({ _id }) => _id).includes(project._id)
+);
```

Comment:
Maybe make this a helper function for all arrays that have objects with `{_id}` , like `containsId(arr, id)` and re use it in containers.